src/App.jsx

```diff
@@ -9,32 +12,27 @@ export const tabs = [
   { id: 'tab-3', title: 'Tab 3', content: 'Some text 3' },
 ];
 
-export const App = () => (
-  <div className="section">
-    <h1 className="title">
-      Selected tab is Tab 1
-    </h1>
-
-    <div data-cy="TabsComponent">
-      <div className="tabs is-boxed">
-        <ul>
-          <li className="is-active" data-cy="Tab">
-            <a href="#tab-1" data-cy="TabLink">Tab 1</a>
-          </li>
+export const App = () => {
+  const [selectedTabId, setSelectedTabId] = useState(tabs[0].id);
 
-          <li data-cy="Tab">
-            <a href="#tab-2" data-cy="TabLink">Tab 2</a>
-          </li>
+  const handleTabToggle = (id) => {
+    setSelectedTabId(id);
+  };
 
-          <li data-cy="Tab">
-            <a href="#tab-3" data-cy="TabLink">Tab 3</a>
-          </li>
-        </ul>
-      </div>
+  return (
+    <div className="section">
+      <h1 className="title">
+        {tabs.map(tab => (
+          `${selectedTabId === tab.id ? `Selected tab is ${tab.title}` : ''}`
+        ))}
+      </h1>
 
-      <div className="block" data-cy="TabContent">
-        Some text 1
-      </div>
+      <Tabs
+        tabs={tabs}
+        selectedTabId={selectedTabId}
+        setSelectedTabId={setSelectedTabId}
+        onTabSelected={handleTabToggle}
```

Comment:
```suggestion
        tabs={tabs}
        selectedTabId={selectedTab.id}
        onTabSelected={handleTabToggle}
```

Reply:
why do you need `setSelectedTabId` ?

Reply:
https://github.com/mate-academy/react_tabs-js/pull/139/files#r1276507731